draft-wang-ppm-dap-taskprov.md

```diff
@@ -579,13 +579,24 @@ Author to be under control of the adversary. It is therefore incumbent on
 protocol participants to verify the privacy parameters of a task before opting
 in.
 
-Another risk is that a malicious coalition of Clients might attempt to pollute
-an Aggregator's long-term storage by uploading reports for many (thousands or
-perhaps millions) of distinct tasks. While this does not directly impact tasks
-used by honest Clients, it does present a Denial-of-Service risk for the
-Aggregators themselves. This can be mitigated by limiting the rate at which new
-tasks or configured. In addition, deployments SHOULD arrange for the Author to
-digitally sign the task configuration so that Clients cannot forge task creation.
+Another risk is that the Author could configure a unique task to fingerprint a
+Client. If the Author colludes with the Leader, it can learn some metadata
+information about the Client, e.g., the Client IP, user agent string, or the
+upload timestamp. However, even though the Author is able to deanonymize the
```

Comment:
The leader already learns the upload timestamp. (It's in the report metadata.)

Also, let's be more precise about how it learns things it's not supposed to.